src/app/routes/utils/fetchPageData/index.js

```diff
@@ -32,10 +32,10 @@ export const getUrl = pathname => {
   return `${baseUrl}${basePath.replace(ampRegex, '')}.json${params}`; // Remove .amp at the end of pathnames for AMP pages.
 };
 
-export default async pathname => {
+export default async (pathname, pageType) => {
```

Comment:
what do you think about making this `({ pathname, pageType })`?

Reply:
I had started down that road - and noticed it would affect 15+ files. Happy to make the changes though - it's my preference too.